cookbook/demo/run.py

```diff
@@ -20,6 +21,7 @@
     agents=[agno_assist, web_search_agent],
     teams=[reasoning_finance_team, multilingual_team],
     workflows=[research_workflow, investment_workflow],
+    knowledge_bases=[standalone_knowledge],
```

Comment:
- We can call the param `knowledge`. It's declarative enough plus it keeps the convention set by the Agent etc classes
- Can we call the instance in this example `knowledge_base` or so? To make it clear it's nothing special, just a common knowledge base